v0x01.py

```diff
@@ -0,0 +1,73 @@
+"""Flow serializer for OF 1.0."""
+from pyof.v0x01.common.action import ActionOutput, ActionType, ActionVlanVid
+from pyof.v0x01.controller2switch.flow_mod import FlowMod
+
+from napps.kytos.flow_manager.main import FlowSerializer as Base
```

Comment:
Same as above for the NApp name.
Even with that fixed, this import is not working (probably related to some relative import constraints: http://bit.ly/2g93eDE )